src/popup.html

```diff
@@ -8,6 +8,8 @@
     <h1>Serenade for Chrome</h1>
     <a href="https://serenade.ai/docs/chrome/"> Read the docs </a>
     <br />
+    <a id="reconnect">Reconnect</a>
```

Comment:
best practice here is either to use a `button` element or set `href="#"`, then call `event.preventDefault` in the listener.